assets/tests/components/nav/bottomNavMobile.test.tsx

```diff
@@ -69,6 +69,6 @@ describe("BottomNavMobile", () => {
       </BrowserRouter>
     )
 
-    expect(screen.queryByTitle("Detours")).toBeNull()
+    expect(screen.queryByTitle("Detours")).toBeInTheDocument()
```

Comment:
the test name seems to be incorrect now?

Reply:
[Touché](https://github.com/mbta/skate/pull/2712/commits/109ed3601a282670b060df99d84be1c18598664e)

Reply:
[Touché part 2](https://github.com/mbta/skate/pull/2712/commits/85a0628011181cf1251ce21ea782984944b8aecb)

(Twoché?)